### ISSUE TITLE
installation is failed on Embulk 0.8.10

### ISSUE THREAD

**Issue by runtakun** [embulk/embulk-output-bigquery#55]
Installation is failed on latest version (0.8.10) of Embulk. I'm not familiar with Ruby, so I just report.

The procedure is as following:

```
# curl -o /usr/local/bin/embulk --create-dirs -L "http://dl.embulk.org/embulk-latest.jar
# chmod +x /usr/local/bin/embulk
# embulk gem install embulk-output-bigquery

Embulk v0.8.10
Successfully installed uber-0.0.15
Successfully installed representable-2.3.0
Successfully installed retriable-2.1.0
Successfully installed addressable-2.4.0
Successfully installed mime-types-data-3.2016.0521
Successfully installed mime-types-3.1
Successfully installed hurley-0.2
Successfully installed multipart-post-2.0.0
Successfully installed faraday-0.9.2
Successfully installed little-plugger-1.1.4
Successfully installed multi_json-1.12.1
Successfully installed logging-2.1.0
Successfully installed jwt-1.5.4
Successfully installed memoist-0.14.0
Successfully installed os-0.9.6
Successfully installed signet-0.7.3
Successfully installed googleauth-0.5.1
ERROR:  While executing gem ... (NoMethodError)
    undefined method `jar_path' for Gem:Module
```

The build environment is:

```
# java -version

openjdk version "1.8.0_91"
OpenJDK Runtime Environment (build 1.8.0_91-8u91-b14-1~bpo8+1-b14)
OpenJDK 64-Bit Server VM (build 25.91-b14, mixed mode)
```

This Dockerfile will reproduces this error:
https://github.com/runtakun/embulk_docker_test

Thanks,

**Comment by runtakun** [comment]
In addition, I could install successfully when I use previous version of Embulk (0.8.9).

**Comment by sonots** [comment]
Could you report this problem to emublk itself?

**Comment by runtakun** [comment]
@sonots Yes, sure!

**Comment by hiroyuki-sato** [comment]
It seems that httpclient related problem.

```
embulk gem install httpclient
2016-07-23 19:06:16.373 +0900: Embulk v0.8.10
Fetching: httpclient-2.8.0.gem (100%)
ERROR:  While executing gem ... (NoMethodError)
    undefined method `jar_path' for Gem:Module
```

**Comment by hiroyuki-sato** [comment]
Raise error here.

https://github.com/embulk/embulk/blob/master/lib/embulk/command/embulk_run.rb#L344

``` ruby
        "java -cp #{jar_path(jar_uri)} org.jruby.Main"
```

``` ruby
  def self.fix_gem_ruby_path
    # Oveerride Gem.ruby method so that rubygems can execute ruby command (as another process).
    # By default, Gem.ruby uses "java -jar jar_path" command. However, it doesn't work because
    # Main-Class is overwritten to org.embulk.cli.Main. Here fixes it to use
    # "java -cp jar_path org.jruby.main" instead.

    jar, resource = __FILE__.split("!", 2)
    jar_uri = URI.parse(jar).path rescue jar

    manifest = File.read("#{jar_uri}!/META-INF/MANIFEST.MF") rescue ""
    m = /Main-Class: ([^\r\n]+)/.match(manifest)
    if m && m[1] != "org.jruby.Main"
      # Main-Class is not jruby
      Gem.define_singleton_method(:ruby) do
        "java -cp #{jar_path(jar_uri)} org.jruby.Main"
      end
    end
  end
```

**Comment by hiroyuki-sato** [comment]
`jar_path` method defined in jruby-9.0.4.0

https://github.com/jruby/jruby/blob/b9fb7aaad77872f08bad7e64ecfa4c481aa3289e/lib/ruby/stdlib/rubygems/defaults/jruby.rb#L34-L39

But not defined in latest jruby(9.1.2.0)
https://github.com/jruby/jruby/blob/master/lib/ruby/stdlib/rubygems/defaults/jruby.rb

**Comment by hiroyuki-sato** [comment]
Maybe, this PR will fix this issue.
https://github.com/embulk/embulk/pull/451

**Comment by sonots** [comment]
fixed with embulk 0.8.11
